loguru_mypy/__init__.py

```diff
@@ -73,6 +75,14 @@ def _loguru_logger_call_handler(
     except IndexError:
         call_kwargs = {}
 
+    call_expr = CallExpr(
+        callee=MemberExpr(expr=log_msg_expr, name="format"),
+        args=call_args,
+        arg_kinds=ctx.arg_kinds[1],
+        arg_names=ctx.arg_names[1],
+    )
+    ctx.api.expr_checker.visit_call_expr(call_expr)
```

Comment:
Now, I know the fact that `ctx.api` is an instance of `TypeChecker` is just a implementation detail, and this isn't part of mypy's plugin API that you can access the `expr_checker` there (and mypy is not happy with that as the `CheckerPluginInterface` doesn't expose that, but is really quite convenient to just be able to hook into mypy's existing methods instead reinventing the wheel and parsing this all by ourselves.

Reply:
I am pretty sure I have stumbled onto these on my own several times. I do not mind that as long as we keep a close eye on that. I am pretty sure I need to finally find time and start testing against several latest releases of mypy + pin the latest on plugin hook.

Reply:
https://github.com/kornicameister/loguru-mypy/issues/47

Reply:
I think that answers your question below about `mypy` version.